src/OnboardingSPA/components/ColorPickerButton/index.js

```diff
@@ -0,0 +1,33 @@
+// This is an Color Picker Option with a color displayed infront of the name
+const ColorPickerButton = ( {
+	isColorSelected,
+	color,
+	slug,
+	name,
+	callback,
+} ) => {
+	return (
+		<div
+			className="custom-palette__below-row"
```

Comment:
```suggestion
			className="color-picker-button__row"
```
Similarly for the other ones.